Update visibility-modifiers.md (#1529)
JetBrains/kotlin-web-site: pages/docs/reference/visibility-modifiers.md

There are four visibility modifiers in Kotlin: `private`, `protected`, `internal` and `public`.
The default visibility, used if there is no explicit modifier, is `public`.

> Please find, below, explanations of how the modifiers apply to different types of declaring scopes.

Reviewer: ```suggestion
On this page, you'll learn how the modifiers apply to different types of declaring scopes.
```
